pkg/version/version_parser.go

```diff
@@ -20,6 +20,7 @@ const preReleaseSeparator = "-"
 const metadataSeparator = "+"
 const prereleaseTokenSeparator = "."
 const snapshotPrereleaseToken = "SNAPSHOT"
+const isIndependentReleaseFormat = `^build\d{12}`
```

Comment:
maybe change this into 
```suggestion
const IndependentReleaseBuildMetadataFormat= `build\d{12}`
```
Keep in mind that the independent version build metadata may not be the only metadata or even the first one...

Reply:
yeah but IAR specs states format strictly like this, version followed by this